### DISCUSSION TITLE
[Feature Request] Enable dynamic selection/deselection navigation

### DISCUSSION THREAD

**Discussion by lucaswhornsby** [grafana/grafana#58491] [Dashboards, Panels & Visualizations]
Currently, as filters are selected on a dashboard – whether by editing the global dropdown filters at the top of the dashboard or clicking on particular values in different panels – it is difficult for the user to keep track of what filters have been applied and to remove those filters incrementally / walk back each step of their navigation if they choose to. Enabling this would provide a much faster and more impactful user experience, since the user could explore different pivots as they are trying to address a question or complete an investigation on a dashboard then easily tweak their filtering without having to reset the dashboard altogether.

We would like to be able to easily track our filters as we select and deselect multiple values/attributes as filters. See this screen recording from a PowerBI dashboard for an example of this visible selection tracking in a multi-value pie chart (not captured in the video – all the correlated dashboards to the right are updating accordingly as filters are selected and deselected).

[Selection and deselection of filters.webm](https://user-images.githubusercontent.com/113460677/200703958-504f9784-7828-47ad-83f0-e0678e1f6993.webm)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
